src/include/duckdb/function/scalar/sequence_functions.hpp

```diff
@@ -8,12 +8,34 @@
 
 #pragma once
 
+#include "duckdb/catalog/catalog_entry/sequence_catalog_entry.hpp"
 #include "duckdb/function/scalar_function.hpp"
 #include "duckdb/function/function_set.hpp"
 #include "duckdb/function/built_in_functions.hpp"
 
 namespace duckdb {
 
+struct NextvalBindData : public FunctionData {
+	explicit NextvalBindData(optional_ptr<SequenceCatalogEntry> sequence)
+	    : sequence(sequence), create_info(sequence ? sequence->GetInfo() : nullptr) {
+	}
+
+	//! The sequence to use for the nextval computation; only if the sequence is a constant
+	optional_ptr<SequenceCatalogEntry> sequence;
+
+	//! The CreateInfo for the above sequence, if it exists
+	unique_ptr<CreateInfo> create_info;
```

Comment:
Is this property necessary here? Can't we instead fetch this when `Serialize` is called?

Reply:
We currently rely on the ability to override the catalog name in the create_info before sending it over the wire (we do a similar thing to the info in LogicalCreateTable / LogicalSimple / LogicalCreate). The reason is that the virtual catalog on the client-side is using a different catalog name than what's used on the server-side (globally unique FQN names for the DBs), and rebinding happens on deserialization.

I understand that it looks a bit odd, but I haven't found a better way to do this.